quantum/quantum.h

```diff
@@ -240,6 +240,10 @@ extern layer_state_t layer_state;
 #    include "os_detection.h"
 #endif
 
+#ifdef LAYER_LOCK_ENABLE
+#    include "process_layer_lock.h"
```

Comment:
To expose the Layer Lock functions like `is_layer_locked()`, include here also

```c
#include "layer_lock.h"
```

Reply:
```suggestion
#    include "layer_lock.h"
```